src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -493,7 +495,10 @@ case class ProjectRecord(
       exports.toList.map { ms =>
         ("exports", exportsDoc(ms)) },
       exclude.toList.map { ms =>
-        ("exclude", exportsDoc(ms)) })
+        ("exclude", exportsDoc(ms)) },
+      processorClasses.toList.map { pcs =>
+        ("processorClasses", list(pcs.toList.sortBy(_.asString)) { pc => quoteDoc(pc.asString) }) }
```

Comment:
if you do `pcs.map(_.asString)` first you can avoid calling it twice, e.g. `list(pcs.map(_.asString).toList.sort)(quoteDoc)`

Reply:
Good idea, thanks.